static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -0,0 +1,165 @@
+import styled from '@emotion/styled';
+
+import Duration from 'sentry/components/duration';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import UserBadge from 'sentry/components/idBadge/userBadge';
+import Link from 'sentry/components/links/link';
+import {StringWalker} from 'sentry/components/replays/walker/urlWalker';
+import ScoreBar from 'sentry/components/scoreBar';
+import TimeSince from 'sentry/components/timeSince';
+import CHART_PALETTE from 'sentry/constants/chartPalette';
+import {IconCalendar} from 'sentry/icons';
+import space from 'sentry/styles/space';
+import type {Organization} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import {spanOperationRelativeBreakdownRenderer} from 'sentry/utils/discover/fieldRenderers';
+import {useLocation} from 'sentry/utils/useLocation';
+import useProjects from 'sentry/utils/useProjects';
+import type {ReplayListRecordWithTx} from 'sentry/views/performance/transactionSummary/transactionReplays/useReplaysFromTransaction';
+import type {VisibleColumns} from 'sentry/views/replays/replayTable/types';
+import type {ReplayListRecord} from 'sentry/views/replays/types';
+
+type Props = {
+  column: keyof VisibleColumns;
+  eventView: EventView;
+  organization: Organization;
+  referrer: string;
+  replay: ReplayListRecord | ReplayListRecordWithTx;
+};
+
+function TableCell({column, eventView, organization, referrer, replay}: Props) {
+  const location = useLocation();
+  const {projects} = useProjects();
+  const project = projects.find(p => p.id === replay.projectId);
+
+  switch (column) {
```

Comment:
Feels like each of these column types should be moved to a subcomponent

Reply:
yeah... the list inside of replayTable.tsx is kind of calling for that too with all the `column="foo"` props

Reply:
changed!